### DISCUSSION TITLE
vyper-debug

### DISCUSSION THREAD

**Discussion by matthewtuby** [vyperlang/vyper#2255] [Q&A] [answered]
Is vdb a dead project? Fixed a (~3) few issues, but they keep comin'. 
Absolutely loved the idea of vyper-run file 'funcition()' -i params for fast prototyping.
Are there any other options for quick executing of functions besides vdb?

**Comment by fubuloubu** [comment, marked as answer]
We'd welcome anyone who wants to take on the responsibility of brining vdb back up to date. I will review any PRs you have to that effect if you need the help.

I'm not aware of any other projects like this, obviously a full debugger is on the wish list of many a web3 dev, and something that would be a nice additional brownie if you're intetrested in contributing: https://github.com/eth-brownie/brownie/issues/151

**Reply by matthewtuby** [reply]
Thank you very much, I'll look into brownie, and work a bit more on vdb, see if i can get it alive. No promises though

**Reply by jacqueswww** [reply]
vyper-debug definitively needs some love! :)
